src/WebUI/WorkLoads/WorkloadDetails.tsx

```diff
@@ -97,7 +111,7 @@ export class WorkloadDetails extends BaseComponent<IWorkloadDetailsProperties, I
 
             return (
                 <div className="content-main-heading">
```

Comment:
change this as per other pr in which customheader is used, so we do not need to adjust the spacing

Your code:
<div className="content-main-heading">
                    <ResourceStatus statusProps={this.props.statusProps} customDescription={headerItem} statusSize={StatusSize.l} className={"w-details-status-header"}/>	                    <ResourceStatus statusProps={this.props.statusProps} customDescription={headerItem} statusSize={StatusSize.l} className={"w-details-status-header"} />

Reply:
Will revisit this once your PR is checked in